docs/howto/metrics/ref_what-are-metrics.adoc

```diff
@@ -12,7 +12,7 @@ To make solving problems easier, applications must expose machine-readable data
 These kinds of data are referred to as _metrics_.
 Collecting metrics, visualizing them, setting alerts, discovering trends, etc. are very important to keep a service healthy.
 
-{Thorntail} provides a fraction for MicroProfile Metrics, an easy-to-use API for exposing metrics.
+{Thorntail} provides a fraction for Eclipse MicroProfile Metrics, an easy-to-use API for exposing metrics.
```

Comment:
This one is good.

Reply:
This is the one at the beginning of the Metrics guide.
That justifies removing the ones in the lower sections of the guide.